src/brevitas/graph/gpxq.py

```diff
@@ -182,6 +182,14 @@ def process_input(self, inp):
                     signed=inp.signed,
                     training=inp.training)
             inp = inp.value
+        elif self.layer.is_input_quant_enabled:
+            self.quant_input = QuantTensor(
+                value=None,
```

Comment:
Instead of None, create a empty tensor (torch.empty), with size 1, `dtype` and `device` equal to the one of the layer's parameters